src/routes.jsx

```diff
@@ -0,0 +1,319 @@
+        return {
+          Component: SkillsQuizPage,
+        };
+      },
+      errorElement: (
+        <RouteErrorBoundary
+          showSiteHeader={false}
+          showSiteFooter={false}
+        />
+      ),
+    },
+    {
+      path: ':courseType?/course/:courseKey',
+      lazy: async () => {
+        const { CoursePage, makeCourseLoader } = await import('./components/course');
+        return {
+          Component: CoursePage,
+          loader: getRouteLoader(makeCourseLoader, queryClient),
+        };
+      },
+      errorElement: (
+        <RouteErrorBoundary
+          showSiteHeader={false}
+          showSiteFooter={false}
+        />
+      ),
+      children: [
+        {
+          index: true,
+          lazy: async () => {
+            const { default: CourseAbout } = await import('./components/course/routes/CourseAbout');
+            return {
+              Component: CourseAbout,
+            };
+          },
+        },
+        {
+          path: 'enroll/:courseRunKey',
+          element: <Outlet />,
+          children: [
+            {
+              index: true,
+              lazy: async () => {
+                const {
+                  default: ExternalCourseEnrollment,
+                  makeExternalCourseEnrollmentLoader,
+                } = await import('./components/course/routes/ExternalCourseEnrollment');
+                return {
+                  Component: ExternalCourseEnrollment,
+                  loader: getRouteLoader(makeExternalCourseEnrollmentLoader, queryClient),
+                };
+              },
+            },
+            {
+              path: 'complete',
+              lazy: async () => {
+                const { default: ExternalCourseEnrollmentConfirmation } = await import('./components/course/routes/ExternalCourseEnrollmentConfirmation');
+                return {
+                  Component: ExternalCourseEnrollmentConfirmation,
+                };
+              },
+            },
+          ],
+        },
+      ],
+    },
+    {
+      path: 'licenses/:activationKey/activate',
+      lazy: async () => {
+        const { default: LicenseActivationRoute } = await import('./components/app/routes/LicenseActivationRoute');
+        return {
+          Component: LicenseActivationRoute,
+        };
+      },
+      errorElement: (
+        <RouteErrorBoundary
+          showSiteHeader={false}
+          showSiteFooter={false}
+        />
+      ),
+    },
+    {
+      path: 'videos/:videoUUID',
+      lazy: async () => {
+        const { makeVideosLoader, VideoDetailPage } = await import('./components/microlearning');
+        return {
+          Component: VideoDetailPage,
+          loader: makeVideosLoader(queryClient),
+        };
+      },
+      errorElement: (
+        <RouteErrorBoundary
+          showSiteHeader={false}
+          showSiteFooter={false}
+        />
+      ),
+    },
+  ];
+  if (hasNotFoundRoutes) {
```

Comment:
[inform] need to sanity check if the two conditional `NotFoundPage` routes (based on `hasNotFoundRoutes`) are still needed now that the approach has changed slightly.

Hunch is these two routes can safely always be included now, simplifying this logic slightly.